frontend/pages/login.vue

```diff
@@ -102,6 +110,20 @@ export default {
           : this.error;
       }
     },
+    authToken() {
```

Comment:
In the case that we expect the url to be non-reactive we can not use a computed property and instead use `created`